### PR TITLE
Add update OAuth Application instructions to Cluster Setup page

### PR DESCRIPTION
**Pull request by tkyi**
After deploying Screwdriver, the OAuth Application needs to be updated with the new Load Balancer Ingress

### CODE REVIEW

File: docs/getting-started/cluster-setup.md [hunk 1]
```diff
@@ -158,3 +159,12 @@ To view the stdout / stderr from a pod, run:
...
 ```
+
+## Update your OAuth Application
+You will need to navigate back to your original OAuth Application that you used for your OAuth Client ID and Secret to update the URLs.
+
+1. Navigate to the [OAuth applications](https://github.com/settings/developers) page.
+
+2. Click on the application you created to get your OAuth Client ID and Secret.
+
+3. Fill out the `Homepage URL` and `Authorization callback URL` with your `LoadBalancer Ingress`.
```
Comment (nkatzman) [inline docs/getting-started/cluster-setup.md:170]: Is it just the `http://load-balancer.aws.com` , or is there a specific route to hit?

Comment (tkyi) [reply docs/getting-started/cluster-setup.md:170]: I believe it's just `http://load-balancer.aws.com`

Comment (nkatzman) [reply docs/getting-started/cluster-setup.md:170]: I think you're right 👍

**Comment by nkatzman** [comment]
👍

**Pull request merged**